nodes/logic/logic_node.py

```diff
@@ -118,13 +118,47 @@ def rclick_menu(self, context, layout):
         layout.prop(self, "output_numpy", expand=False)
 
     def migrate_from(self, old_node):
```

Comment:
I think I would prefer like this:
```py
def migrate_from(self, old_node):
    if old_node.bl_idname == 'SvLogicNodeMK2':
        return
```

Reply:
Ok, fixed.

Reply:
I meant that this is the only changes which are required for the node to work properly.